spec/directions/bake_footnotes/v1_spec.rb

```diff
@@ -51,37 +51,37 @@
     expect(book1.body.children.to_s).to match_normalized_html(
       <<~HTML
         <div data-type="page">
-          <p><a href="#aside1" role="doc-noteref">1</a> Blah.</p>
+          <p class="has-noteref"><a href="#aside1" role="doc-noteref">1</a> Blah.</p>
```

Comment:
Will this add something to past books?

Reply:
I had the same question, I'll put a thread in the channel for opinions